qc_schema/dev/molecule.py

```diff
@@ -0,0 +1,99 @@
+"""
+The json-schema for the Molecule definition
+"""
+molecule = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "properties": {
+        "symbols": {
+            "description": "The atom symbol for each atom in the molecule.",
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        },
+        "geometry": {
+            "description": "The (3N, ) vector of XYZ coordinates of the atoms.",
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "masses": {
+            "description": "The masses of the atoms in the molecule, canonical weights assumed if not given.",
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "name": {
+            "description": "The name of the molecule.",
+            "type": "string"
+        },
+        "comment": {
```

Comment:
Let's make these part of an object for "identifiers" to reflect feedback from #35